apps/condo/pages/ticket/index.tsx

```diff
@@ -987,7 +987,10 @@ const TicketsPage: ITicketIndexPage = () => {
                     <FavoriteTicketsContextProvider
                         extraTicketsQuery={{ ...ticketFilterQuery, organization: { id: userOrganizationId } }}
                     >
-                        <Row gutter={breakpoints.DESKTOP_LARGE && MEDIUM_VERTICAL_ROW_GUTTER} style={MAIN_ROW_STYLE}>
+                        <Row
+                            gutter={breakpoints.DESKTOP_LARGE && MEDIUM_VERTICAL_ROW_GUTTER}
```

Comment:
Other sizes don't need "gutter"?

Reply:
on mobile screens has enough indentation at the title